exonum/src/runtime/rust/interfaces.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Exonum Team
```

Comment:
Is Initialize trait Rust-specific? If not, why is it placed inside `rust` module?

Reply:
Yes this trait is Rust specific and you have to implement in Java same interface without binding to Rust.

Reply:
So than interface name and methods ids must be available in the `runtime` module, as well as the list of the interfaces which must be supported by the runtime implementation.